app/src/main/java/com/omen273/crossLingo/GameActivity.kt

```diff
@@ -53,18 +50,32 @@ import android.widget.Toast
 
 import android.content.Intent
 import android.graphics.Typeface
+import android.view.*
 import android.view.inputmethod.EditorInfo
 import android.widget.TextView
 import android.widget.EditText
-
+import androidx.core.graphics.drawable.DrawableCompat
 
 class GameActivity : AppCompatActivity(), CrosswordView.OnLongPressListener,
-    CrosswordView.OnStateChangeListener, CrosswordView.OnSelectionChangeListener, CrosswordView.OnPrintLetterListener {
+    CrosswordView.OnStateChangeListener, CrosswordView.OnSelectionChangeListener,
+    CrosswordView.OnPrintLetterListener,
+    CrosswordView.OnWordSolved {
 
     private lateinit var crosswordView: CrosswordView
     internal var name = ""
     private var delete = false
     private var onBackPressedCallBefore = false
+    private lateinit var cellMenuItem: MenuItem
+    private lateinit var wordMenuItem: MenuItem
```

Comment:
It should be parceable

Reply:
It should not be parceable because these variables are recreated during OnCreate